project/dexter-peer-review/src/test/com/samsung/sec/dexter/peerreview/plugin/DexterPeerReviewPlugInTest.java

```diff
@@ -59,4 +59,51 @@ private AnalysisConfig createTestConfig() {
         return analysisConfig;
 	}
 
+	@Test
+	public void test_getAllCommentFromSourcecode_basic_success1(){
```

Comment:
What is different between the test case '_success1' and '_seccess2'.
It would be better to write the test spec in more detail.(ex: oneLineComment or multiLineComment)

and it's better to add the test cases for 'MAJ', 'CRI' comments.